### ISSUE TITLE
Suggestion: No friendly fire (with mangonel)

### ISSUE THREAD

**Issue by realPhint** [Sh0wdown/UnofficialCrusaderPatch#602]
As the title says, maybe this could be an option. So basically when your mangonel hits your own units/buildings or those of allies, the stones would just dissapear without causing any harm. For the AI this is already the case.
Perhaps this option could also be expanded to prevent any sort of friendly fire.

thank you & best regards
Phint

**Comment by LordHansCapon** [comment]
Yes, probably possible, but its more like a game balancing feature, so you don't just spam mangonels, and you have to control these units like your catapults to not hit your own units. On Discord, we have talked about whether we should implement such optional changes which can de-balance the game (like if its balanced...), and we mostly agreed on not to de-balance the game hugely. Now what counts as a huge de-balance I am not sure.

For some, it is necessary not to de-balance it because of the multiplayer aspect, but for players like myself, who only play singleplayer it doesn't matter, if one wants to build mangonels on smaller towers or disable friendly fire on mangonels, its their choice I guess.

As for multiplayer, most of the stuff desyncs or flat on crashes multiplayer if not all the players have the same configuration in the UCP. So agreeing on one config in a multiplayer environment is more than acceptable, mostly required.
